tests/subsys/caf/sensor_manager/configuration/common/sensor_sim_ctrl_def.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "sensor_sim_ctrl.h"
+
+/* This configuration file is included only once from sensor_sim_ctrl module and holds
+ * information about wave signals that should be generated by simulated sensor.
+ */
+
+/* This structure enforces the header file is included only once in the build.
```

Comment:
This phase is also in many other places. I think if we want to change it it would be good to create separate PR for that. @divipillai should we do it?

Reply:
@zycz :  I think we can make changes to this particular header file as part of this PR, and for the rest of the files, we can fix it as and when we touch that particular header file.

Reply:
So you suggest not to change all other files? Those header file are not touched very often and I don't think anyone would remember about making that change when some change will be introduced.

Reply:
As this is a code comment, I think we can ignore this for now.